src/MCPClient/tests/test_antivirus.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf8 -*-
+"""Tests for the archivematicaClamscan.py client script."""
+
+import os
+import sys
+
+from collections import OrderedDict, namedtuple
+
+import pytest
+import test_clamdscan
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(
+    os.path.abspath(os.path.join(THIS_DIR, '../lib/clientScripts')))
+
+import archivematicaClamscan
+
+# Antivirus Tests
+
+
+def test_get_scanner(settings):
+    """ Test that get_scanner returns the correct instance of antivirus
+    per the user's configuration. Test return of clamdscanner by default. """
+
+    # doubly ensure that clamd env settings are available to the test
```

Comment:
Comments should be sentences: https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html (from the guy who wrote coverage.py).

Reply:
Resolved.